### PR TITLE
Issue 119281: Extract table from daviz links and animation effect on view

### PR DESCRIPTION
**Pull request by nileshgulia1**

### CODE REVIEW

**Review by tiberiuichim — CHANGES_REQUESTED** [review]
That's not gonna work too great, at least the port is always 3000 in deployment.

I don't know if you need the getParsedSVG action anymore, volto-corsproxy includes an action that you can use that will get you your content in the store.content reducer. See volto-corsproxy documentation on that.

**Comment by nileshgulia1** [comment]
@tiberiuichim I have used `getProxiedExternalContent` instead of new actions. However what I found is sometimes it does not fetch the content from external site and gives 404 whereas most of the time it works but we need to handle the error condition also.

File: src/components/manage/Blocks/DataFigure/Svg.jsx [hunk 1]
```diff
@@ -23,18 +22,12 @@ import {
...
-    ? `http://${host}:${port}`
-    : `${window.location.protocol}//${window.location.host}`;
-
-  const path = `${base}${corsProxyPath}/${data.url}`;
 
   React.useEffect(() => {
     if (data.url.includes('.svg')) {
       if (!isInternalURL(data.url)) {
-        dispatch(getSVG(path))
+        dispatch(getProxiedExternalContent(data.url, { headers: { Accept: 'image/svg+xml', } }))
```
Comment (tiberiuichim) [inline src/components/manage/Blocks/DataFigure/Svg.jsx:30]: Could you please tweak the volto-corsproxy to mixin the request headers? 

In theory, by default, it's really important that we don't pass the authentication headers, and I don't know if, if you overwrite the headers, if the authentication headers are not still sent.

**Review by tiberiuichim — CHANGES_REQUESTED** [review]

File: src/components/manage/Blocks/DataFigure/Svg.jsx (hunk 1 above)
Comment (nileshgulia1) [reply src/components/manage/Blocks/DataFigure/Svg.jsx:30]: Done!

Comment (nileshgulia1) [reply src/components/manage/Blocks/DataFigure/Svg.jsx:30]: https://github.com/eea/volto-corsproxy/pull/1

Comment (tiberiuichim) [reply src/components/manage/Blocks/DataFigure/Svg.jsx:30]: You should still keep `...request` in the action in corsproxy

**Review by avoinea — APPROVED** [review]

**Pull request merged**